WalletWasabi.Gui/MainWindow.xaml.cs

```diff
@@ -153,6 +153,7 @@ private async Task ClosingAsync()
 							Global.UiConfig.WindowState = WindowState;
 							Global.UiConfig.Width = Width;
 							Global.UiConfig.Height = Height;
+							Global.UiConfig.Tab = IoC.Get<IShell>().SelectedDocument.Title;
```

Comment:
In case there is no open tab, you will get a `NullReferenceException` here. Anyway, I don't think this is the best way to implement it. I think every time you select a tab it should be saved in the UiConfig.json file and not when the user closes the application.

Reply:
Btw, I can confirm there is a NRE. After that wasabi UI crashes and doesn't close (black window), you have to kill the process.

Reply:
Fixed.

Reply:
>  I think every time you select a tab it should be saved in the UiConfig.json file and not when the user closes the application.

That is not necessary.
Imagine if I keep jumping from a tab to another and then close Wasabi, then the last one will be saved, so it is the same result with more work.